mail-function/package.json

```
@@ -4,15 +4,30 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "exec": "src/index.js"
+    "exec": "src/index.js",
+    "eslint": "eslint '**/*.{js,jsx,ts,tsx}' && tsc",
```

```suggestion
    "eslint": "eslint '**/*.{js,jsx,ts,tsx}'",
```
There is no typescript, so you don't have a typescript compiler (tsc)